tests/unit/test_error_handling.py

```diff
@@ -349,6 +357,52 @@ def test_automatic_tap_changing():
     model.calculate_power_flow(tap_changing_strategy=TapChangingStrategy.min_voltage_tap)
 
 
+def test_conflicting_angle_measurement_type():
+    node_input = initialize_array("input", "node", 2)
+    node_input["id"] = [0, 1]
+    node_input["u_rated"] = [10e3, 10e3]
+
+    line_input = initialize_array("input", "line", 1)
+    line_input["id"] = [2]
+    line_input["from_node"] = [0]
+    line_input["to_node"] = [1]
+
+    source_input = initialize_array("input", "source", 1)
+    source_input["id"] = [3]
+    source_input["node"] = [0]
+    source_input["status"] = [1]
+    source_input["u_ref"] = [1.0]
+
+    sym_current_sensor_input = initialize_array("input", "sym_current_sensor", 2)
+    sym_current_sensor_input["id"] = [4, 5]
+    sym_current_sensor_input["measured_object"] = [2, 2]
+    sym_current_sensor_input["measured_terminal_type"] = [
+        MeasuredTerminalType.branch_from,
+        MeasuredTerminalType.branch_from,
+    ]
+    sym_current_sensor_input["angle_measurement_type"] = [
+        AngleMeasurementType.local_angle,
+        AngleMeasurementType.global_angle,
+    ]
+    sym_current_sensor_input["i_sigma"] = [1.0, 1.0]
+    sym_current_sensor_input["i_angle_sigma"] = [1.0, 1.0]
+    sym_current_sensor_input["i_measured"] = [0.0, 0.0]
+    sym_current_sensor_input["i_angle_measured"] = [0.0, 0.0]
+    repr(sym_current_sensor_input)
+
+    model = PowerGridModel(
+        input_data={
+            "node": node_input,
+            "line": line_input,
+            "source": source_input,
+            "sym_current_sensor": sym_current_sensor_input,
```

Comment:
Not that it should affect the test, but can you also add one voltage sensor without angle measurement?